tests/core/test_ensemble.py

```diff
@@ -132,8 +132,8 @@ def test_policy_priority():
 def test_fallback_mapping_restart():
     domain = Domain.load("data/test_domains/default.yml")
     events = [
-        ActionExecuted(ACTION_DEFAULT_FALLBACK_NAME),
-        utilities.user_uttered(USER_INTENT_RESTART, 1),
+        ActionExecuted(ACTION_DEFAULT_FALLBACK_NAME, timestamp=1),
+        utilities.user_uttered(USER_INTENT_RESTART, 1, timestamp=2),
```

Comment:
is this required for windows tests to pass or just a general fix?

Reply:
It's required. For some reason the timestamps on Windows are the same thus the order in this test cannot be guaranteed.